skimage/feature/corner.py

```diff
@@ -123,25 +123,29 @@ def hessian_matrix(image, sigma=1, mode='constant', cval=0):
     mode : {'constant', 'reflect', 'wrap', 'nearest', 'mirror'}, optional
         How to handle values outside the image borders.
     cval : float, optional
+        else:optional
         Used in conjunction with mode 'constant', the value outside
         the image boundaries.
+    order : {'xy', 'rc'}, optional
+        this parameter allows for the use of reverse or forward order of
+        returned matrix values
```

Comment:
Add proper capitalisation and full stop. Also, I wouldn't call this reverse order of matrix values, but rather, reverse order of image axes in gradient computation. 'rc' indicates first axis first (Hrr), while 'xy' indicates last axis first (Hxx, which is the same as Hcc in 2D). Try to get this across in the docstring (feel free to reuse my text).